tiledb/sm/c_api/tiledb_dimension_label_experimental.h

```diff
@@ -183,6 +183,60 @@ TILEDB_EXPORT capi_return_t tiledb_array_schema_set_dimension_label_tile_extent(
     tiledb_datatype_t type,
     const void* tile_extent) TILEDB_NOEXCEPT;
 
+/**
+ * Retrieve the number of dimension labels attached to an array schema.
+ *
+ * **Example:**
+ *
+ * @code{.c}
+ * uint64_t dim_label_num = 0;
+ * tiledb_array_schema_get_dimension_label_num(
+ *      ctx,
+ *      array_schema,
+ *      &dim_label_num);
+ * @endcode
+ *
+ * @param ctx The TileDB context.
+ * @param array_schema The array schema.
+ * @param dim_label_num Retrieves the number of dimension labels for the schema.
```

Comment:
Nit: Think this was supposed to be `Retrieved` instead of `Retrieves`

Reply:
Oops, thanks! :+1: